src/pcapi/utils/config.py

```diff
@@ -30,7 +30,7 @@
     API_URL = f"https://backend.passculture-{ENV}.beta.gouv.fr"
     API_APPLICATION_NAME = "pass-culture-api-dev"
     WEBAPP_URL = f"https://app.passculture-{ENV}.beta.gouv.fr"
-    PRO_URL = "https://pro.testing.passculture.app"
+    PRO_URL = f"https://pro.passculture-{ENV}.beta.gouv.fr"
```

Comment:
Puisqu'on connaît la valeur de `ENV` dans cette branche : autant l'écrire en toute lettres, ce serait plus lisible, non ?

```suggestion
    PRO_URL = "https://pro.passculture-testing.beta.gouv.fr"
```

(idem pour les autres constantes de ce bloc)

Ou alors on garde les `-{ENV}`, mais on fusionne (si possible) avec le bloc `else` suivant. Car, actuellement, on doit un peu se poser la question de ce qui est différent avec le bloc `else` suivant. :) (Réponse : seul `API_APPLICATION_NAME` est différent.)

Reply:
Ah mince, je suis aller trop vite.
Je suis en phase avec tes remarques, j'en ai parlé avec Cyril en parallèle.

Cette conf n'est pas claire. J'ai poussé en l'état pour débloquer la revue PO mais j'ai prévenu Vincent que j'allais faire un micro ticket de nettoyage dans le code pour virer les reliquats et dans l'infra pour avoir un truc cohérent.

Je prend la remarque dans cette autre PR :)